.enb/make.js

```diff
@@ -41,6 +41,41 @@ isProd || levels.push('development.blocks');
 module.exports = function(config) {
     config.setLanguages(['en', 'ru']);
 
+    config.includeConfig('enb-bem-tmpl-specs'); // Подключаем `enb-bem-tmpl-specs` модуль.
+
+    var examples = config.module('enb-bem-tmpl-specs') // Создаём конфигуратор сетов
+        .createConfigurator('tmpl-specs', {            // в рамках таска `specs`.
+            coverage: {                                // Определяем общие опции для всех уровней-сетов.
+                engines: ['bemhtml'],
+                reportDirectory: 'coverage',
+                exclude: ['**/node_modules/**', '**/libs/**'],
+                reporters: ['lcov']
+            }
+        });
+
+    examples.configure({
+        destPath: 'common.tmpl-specs',
+        levels: ['common.blocks'],
+        langs: true,
+        sourceLevels: levels,
+        engines: {
+            'bemhtml-dev': {
+                tech: 'enb-bemxjst/techs/bemhtml',
+                options: {
+                    exportName: 'BEMHTML',
+                    devMode: true
```

Comment:
remove

add `engineOptions` (?) with the same values as in `techs.bemhtml` bellow